web-api/openapi/front_office.yaml

```diff
@@ -197,6 +223,29 @@ components:
             score: 0.87
             properties:
               title: "News title"
+    SemanticSearchText:
+      type: object
+      required: [text]
+      properties:
+        text:
+          type: string
+          pattern: ".+"
+          minLength: 1
+          maxLength: 250
```

Comment:
:art: we can probably allow for more bytes in the request string. the embedder accepts up to 250 tokens in its default configuration, each token can consist of multiple chars, and each char can consist of up to four bytes. in the nc repo we set this value to 4096, this might be a good value here as well.